lession_10/test_homework_10.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+from unittest.mock import patch
+import sys
+import os
+sys.path.append('/home/bardin/PycharmProjects/My project/AQA-Python-Hillel/lession_10')
+
+from homework_10 import log_event
+
+
+class TestLogEvent(unittest.TestCase):
+
+    @patch('homework_10.logging.getLogger')  # Патчим getLogger
```

Comment:
This solution can be in place. But we need to test real logger work. logger function in another file creates file with logs. Better solution is not to mock it - but to read file and parse from there logs messages as actual result and compare with expected one.